ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/VirtualMachineCvesPage.tsx

```diff
@@ -18,7 +18,7 @@ function VirtualMachineCvesPage() {
             {hasReadAccessForIntegration && <ScannerV4IntegrationBanner />}
             <Routes>
                 <Route index element={<VirtualMachineCvesOverviewPage />} />
-                <Route path=":virtualMachineId" element={<VirtualMachinePage />} />
+                <Route path="virtualmachines/:virtualMachineId" element={<VirtualMachinePage />} />
```

Comment:
What is pro and con of a constant variable as single source of truth for **virtualmachines**:
* For route path here (by the way, hurray for React Router relative paths)
* For `getVirtualMachineEntityPagePath` function

Reply:
> by the way, hurray for React Router relative paths  ➕

I think it makes a lot of sense, the real question imo is where would that variable belong? I would most likely expect to see it here or in some dedicated `vulnerabilities.routes` file? I'll also point out in this particular case we didn't follow that pattern anywhere else in vulnerabilities so I guess that could be a con here.